extern/sector-storage/ffiwrapper/sealer_cgo.go

```diff
@@ -371,7 +371,7 @@ func (sb *Sealer) ReadPiece(ctx context.Context, writer io.Writer, sector abi.Se
 	maxPieceSize := abi.PaddedPieceSize(sb.ssize)
 
 	pf, err := openPartialFile(maxPieceSize, path.Unsealed)
-	if xerrors.Is(err, os.ErrNotExist) {
```

Comment:
This was probably meant to ignore some particular error but agree that we should list it explicitly and otherwise error out by default.